### ISSUE TITLE
More tests

### ISSUE THREAD

**Issue by sindresorhus** [yeoman/insight#2]

**Comment by riyadhalnur** [comment]
@sindresorhus what's the coverage atm?

**Comment by sindresorhus** [comment]
We don't have test coverage set up. The title is misleading. I just meant more tests.

https://github.com/yeoman/insight/blob/master/test/test.js

**Comment by riyadhalnur** [comment]
@sindresorhus yeah makes more sense now. Can you set up a code coverage tool for this then?

**Comment by sindresorhus** [comment]
I don't have the time.
